src/npcmove.cpp

```diff
@@ -1804,15 +1804,15 @@ void npc::move_away_from( const std::vector<sphere> &spheres, bool no_bashing )
     std::vector<tripoint> escape_points;
 
     std::copy_if( range.begin(), range.end(), std::back_inserter( escape_points ),
-    [&]( const tripoint & elem ) {
```

Comment:
Changing of `npcmove.cpp` is not necessary as it brings astyle regression, otherwise looks good.

Reply:
ACK, my astyle version disagrees with the one on Jenkins, so it keeps telling me to change random things like this.